src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,109 @@
 
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+export const FindMovie = ({ newMovies }) => {
+  const [title, setTitle] = useState('');
+  const [preview, setPreview] = useState(movies[0]);
+  const [filmIsExist, setMovieExist] = useState(true);
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+  const findExistMovie = () => {
+    const titleToLow = title.toLowerCase();
+    const moviesFromData = movies.find(movie => movie
+      .title.toLowerCase() === titleToLow);
+
+    if (moviesFromData) {
+      setPreview(moviesFromData);
+    }
+
+    getFilm(title)
+      .then((movie) => {
+        if (movie.Response === 'False') {
+          setMovieExist(false);
+        } else {
+          const moviesFromServer = {
+            title: movie.Title,
+            description: movie.Plot,
+            imdbId: movie.imdbID,
+            imgUrl: movie.Poster,
+            imdbUrl: `https://www.imdb.com/title/${movie.imdbID}`,
+          };
+
+          setPreview(moviesFromServer);
+        }
+      });
+  };
+
+  return (
+    <>
+      <form
+        className="find-movie"
+        onSubmit={(event) => {
+          event.preventDefault();
+          setTitle('');
+          setPreview([movies[0]]);
+        }}
+      >
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={filmIsExist ? 'input' : 'input is-danger'}
+              value={title}
+              onChange={event => setTitle(event.target.value)
+              }
+            />
+          </div>
+          { filmIsExist || (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
+            </p>
+          )
+          }
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
```

Comment:
```suggestion
              type="submit"
```

Reply:
fix this, this button is better to be of type submit